jira/resources.py

```diff
@@ -692,6 +692,15 @@ def update(  # type: ignore[override] # incompatible supertype ignored
 
         super(Issue, self).update(async_=async_, jira=jira, notify=notify, fields=data)
 
+    def get_field(self, field_name: str) -> Any:
+        if field_name.startswith("_"):
+            raise AttributeError(
+                f"A issue field_name cannot start with underscore (_): {item}",
+                field_name,
```

Comment:
```suggestion
                f"A issue field_name cannot start with underscore (_): {field_name}"
```

`item` is not defined

Reply:
This one is embarrassing :cold_sweat:

Reply:
write a test for it :-) then this wont happen.

> does it show I'm a QA person :-)